### PR TITLE
Add scripts to remove bucket websites and omit from pre build flow

### PR DESCRIPTION
**Pull request by davemcorwin**
## Changes proposed in this pull request:
- Addresses #3738 
- Remove adding the website config during the pre-first-build process
- Adds a script to remove website configurations from existing buckets

Operating without the website configurations requires the proxy changes currently in staging to be deployed. To be fully tested in staging.

## Research
Current bucket configurations: https://github.com/raylene-gsa 

## Todo
Remove this script when migration is complete.

## security considerations
None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,7 +94,8 @@
     "migrate-build-logs": "node ./scripts/migrate-build-logs.js",
     "send-email": "node ./scripts/send-email.js",
     "bootstrap-admins": "node ./scripts/bootstrap-admins.js",
-    "migrate-build-notification-settings": "node ./scripts/migrate-build-notification-settings.js"
+    "migrate-build-notification-settings": "node ./scripts/migrate-build-notification-settings.js",
+    "remove-bucket-website-configs": "node ./scripts/remove-bucket-website-configs.js"
```
Comment (svenaas) [inline package.json:98]: I'm not seeing this file.

**Review by svenaas — COMMENTED** [review]
Did you mean to attach `scripts/remove-bucket-website-configs.js`?

File: package.json (hunk 1 above)
Comment (davemcorwin) [reply package.json:98]: yikes, think i lost it when i renamed it, it's baaaaaaaaack

**Review by svenaas — APPROVED** [review]
There are limits to how thoroughly I can verify the implementation at this point, but with the addition of the remove-bucket script there is no longer anything that alarms me. LGTM.

**Comment by davemcorwin** [comment]
thanks! we'll know more once we try to run it in staging :)

**Pull request merged**